src/guides/share/groups/customize.rst

```diff
@@ -24,7 +24,7 @@ You can customize the content of your group's page by uploading two files to the
 
 Create a new file named ``group-overview.md`` that will contain information about your group.
 At the top of this file, provide a title for the page, a list of people who maintain the data, a website, and/or whether to show datasets and narratives from your group.
-This information is technically known as the `YAML front matter <https://jekyllrb.com/docs/front-matter/>`_ for the file.
+This information is technically known as the `YAML front matter <https://jekyllrb.com/docs/front-matter/>`__ for the file.
```

Comment:
I didn't realize there was a significant difference between `_` and `__`. Thanks for pointing it out!